static/app/components/draggableTabs/item.tsx

```diff
@@ -0,0 +1,12 @@
+import {Item as _Item} from '@react-stately/collections';
+import type {ItemProps} from '@react-types/shared';
+import type {LocationDescriptor} from 'history';
+
+export interface DraggableTabListItemProps extends ItemProps<any> {
+  key: string | number;
+  disabled?: boolean;
+  hidden?: boolean;
+  to?: LocationDescriptor;
+}
```

Comment:
This is currently exactly the same as the current TabListItemProps, but there will eventually be new props added that are specific to the draggable version.